src/main/java/com/revealprecision/revealserver/api/v1/dto/factory/EntityTagFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.revealprecision.revealserver.api.v1.dto.factory;
+
+import com.revealprecision.revealserver.api.v1.dto.request.EntityTagRequest;
+import com.revealprecision.revealserver.enums.EntityStatus;
+import com.revealprecision.revealserver.persistence.domain.EntityTag;
+import com.revealprecision.revealserver.persistence.domain.LookupEntityType;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class EntityTagFactory {
+
+  public static EntityTag toEntity(EntityTagRequest entityTagRequest,
+      LookupEntityType lookupEntityType) {
+
```

Comment:
if the request does not have validation, you will get null pointers here , so this code assumes validation has happened on the request. please update the request to support this assumption

Reply:
agree